server/src/db/seeders/20210219184933-contentType.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('contentTypes', [
+      {
+        name: 'capabilities',
+        titleColor: 'green',
+        backgroundColor: 'green',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: 'features',
+        titleColor: 'red',
+        backgroundColor: 'red',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: 'benefits',
+        titleColor: 'red',
+        backgroundColor: 'red',
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      },
+      {
+        name: 'integrated systems',
+        titleColor: 'red',
```

Comment:
my only comment is that you may want to add the id with a uuid from a generate function online. This way it is the same every time as you will need the id for your other seed files in different tables later on

Reply:
thanks - fixed